Vagrantfile

```diff
@@ -13,21 +13,19 @@ Vagrant.configure("2") do |config|
     xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
     
     # Install and configure pyenv
-    sudo rm -r .pyenv
     sudo git clone https://github.com/pyenv/pyenv.git ~/.pyenv
     echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
     echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
     echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(sudo pyenv init -)"\nfi' >> ~/.profile
     cd .pyenv
     sudo mkdir shims
-    sudo chmod 0777 shims
+    sudo chmod 777 shims
     sudo mkdir versions
```

Comment:
I'm curious what these directories fixed with pyenv.

This is the kind of thing that deserves a code comment. Would be more useful than comments like "# Update all packages" above "sudo apt-get update"